config-sample.yaml

```diff
@@ -107,6 +107,17 @@ rtc:
   # # are disabled and clients don't ACK opened peer connections, only reliable, ordered delivery
   # # will be available.
   # strict_acks: true
+  # #
+  # # Settings for improving throughput in high stress forwarding load case.
+  # #
+  # # when set, Livekit will use UDP mux to listen on udp ports (up to runtime.NumCPU()) to
+  # # balance traffic between those ports, also superior to udp_port
```

Comment:
should this say `overrides udp_port`? i. e. when `udp_ports` is set, it will be override `udp_port` if set.